package.json

```diff
@@ -1,15 +1,19 @@
 {
-  "name": "@momento/sdk",
+  "name": "client-sdk-javascript",
```

Comment:
We previously agreed to not use language names in package names. What does a user import from npmjs in their own app with this package name?

Reply:
I wasn't aware of that decision made previously.
The reason why I updated the package name is that with `@momento/sdk`, I was not able to publish to npm.
So I'm more than happy to bring this up in the developer-ecosystem channel to see what name is appropriate.

Reply:
I still am curious, what does a user import from npmjs in their own app with this package name? I think it's going into a shared global namespace. We should look to other sdk vendors for inspiration too.

This is a good slack conversation to have!